client/src/utils_links.js

```diff
@@ -24,7 +24,8 @@ const ENTITY_TYPE_TO_RELATED_OBJECT_TYPE = flip(
 )
 
 export function getEntityInfoFromUrl(url) {
-  const urnMatch = url.match(/^urn:anet:([^:]*):(.*)$/)
+  const urnMatch =
+    url.match(/^urn:anet:([^:]*):(.*)$/) || url.match(/^\/([^.]*)\/(.*)$/)
```

Comment:
The second regex match should probably be: ` url.match(/^\/([^/]*)\/(.*)$/)` i.e. a slash, followed by anything that is not a slash, followed by a slash, followed by anything.